x-pack/plugins/enterprise_search/README.md

```diff
@@ -66,6 +66,83 @@ sh jest.sh public/applications/shared/flash_messages/flash_messages_logic.test.t
 
 ### E2E tests
 
+We currently have two testing libraries in which we run E2E tests:
+
+- [Cypress](#cypress-tests)
+  - Will contain the majority of our happy path E2E testing
+- [Kibana's Functional Test Runner (FTR)](#kibana-ftr-tests)
+  - Contains basic tests that only run when the Enterprise Search host is not configured
+  - It's likely we will not continue to expand these tests, and might even trim some over time (to be replaced by Cypress)
```

Comment:
This will need to be kept up-to-date as we write more Cypress tests and evaluate what we want to do with our currently written FTR tests